src/pages/signin/SignInPageLayout/Footer.js

```diff
@@ -13,16 +14,33 @@ import Licenses from '../Licenses';
 import Socials from '../Socials';
 import Hoverable from '../../../components/Hoverable';
 import CONST from '../../../CONST';
-import Navigation from '../../../libs/Navigation/Navigation';
+import Navigation, {navigationRef} from '../../../libs/Navigation/Navigation';
 import * as Session from '../../../libs/actions/Session';
+import screens from '../../../SCREENS';
 
 const propTypes = {
+    scrollViewRef: PropTypes.shape({
+        // eslint-disable-next-line react/forbid-prop-types
+        current: PropTypes.any,
```

Comment:
@allroundexperts we should find and use the correct proptype instead of `any`

Reply:
Is it same as forwadedRef?
 
```
forwardedRef: PropTypes.oneOfType([
        PropTypes.func,
        PropTypes.shape({current: PropTypes.instanceOf(React.Component)}),
]),
```

Reply:
scrollViewRef: PropTypes.shape({
        current: PropTypes.instanceOf(ScrollView),
    }).isRequired,

Reply:
but again, you said it doesn't work on native?